packages/release-readiness-review/tests/test_review.py

```
@@ -0,0 +1,237 @@
+
+
+def test_timestamp_runtime_and_chronology_fail_closed() -> None:
+    """Human review time is exact UTC and cannot postdate system issuance."""
+    kwargs = valid_kwargs()
+    kwargs["reviewed_at"] = "2020-01-01T00:00:00Z"
+    with pytest.raises(ValueError, match="exact built-in datetime"):
+        build_release_readiness_review_packet(**kwargs)
+
+    kwargs = valid_kwargs()
+    kwargs["reviewed_at"] = datetime(2020, 1, 1)
+    with pytest.raises(ValueError, match="timezone.utc"):
```

Ruff (0.16.3)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)